Package.swift

```diff
@@ -0,0 +1,43 @@
+// swift-tools-version:5.4
+import PackageDescription
+
+let package = Package(
+    name: "xcresultparser",
+    platforms: [
+        .macOS(.v11),
+    ],
+    products: [
+        .executable(
+            name: "xcresultparser",
+            targets: ["xcresultparser"]
+        ),
+    ],
+    dependencies: [
+        .package(
+            name: "swift-argument-parser",
+            url: "https://github.com/apple/swift-argument-parser.git",
+            .exact("0.4.3")
+        ),
+        .package(
+            name: "XCResultKit",
+            url: "https://github.com/davidahouse/XCResultKit.git",
+            .exact("0.9.1")
```

Comment:
XCResultKit doesn't seem to use semantic versioning. They introduced breaking changes from 0.9.1 to 0.9.2. Odd.
I had indeed set it to .upToNextMajor. But I will fix the issues to make it 0.9.2 compatible.